src/components/TodoList.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import Todo from './Todos';
+
+function TodoList ({todos}) {
+  return (
+    <ul>
+      {todos.map(todo => (
+          <li key={todo.id}>
+            <Todo {...todo} />
+          </li>
+      ))}
+    </ul>
```

Comment:
```suggestion
    <ul>
      {todos.map(todo => (
        <li key={todo.id}>
          <Todo {...todo} />
        </li>
      ))}
    </ul>
```
Too many indentations